src/buttons/Button.js

```diff
@@ -66,6 +66,16 @@ class Button extends Component {
       ...attributes
     } = this.props;
 
+    // Refactor to Pressable
+    const TouchableComponentInternal = TouchableComponent
```

Comment:
I think this can be changed to:

```js
const TouchableComponentInternal = TouchableComponent || 
  Platform.select(...);
```

Reply:
@flyingcircle  thanks for the feedback :)